MarvellOS.py

```diff
@@ -389,7 +401,19 @@ def sample(self):
         self.nxt.place(x=280,y=310)
         self.bck=Button(self.samwin,text="<", bg="yellow", font=("Bauhaus 93",20),command=self.bck, state=DISABLED)
         self.bck.place(x=10,y=310)
+
+        self.chooseFile = Button(self.samwin, text ="Select Image",command =  lambda: self.open_file(), padx = 10,pady = 10, font=("Bauhaus 15 ",10))
```

Comment:
You can change the button to "Upload Image" after you implement the image saving thing.